### PR TITLE
Feature/organize packages

### LINKED ISSUE #2810: Move all APInf packages' into /apinf_packages directory
Our project structure is currently very flat, with the top level containing many directories. This can be a bit disorienting when first glancing the source code.

# Goal
Make the source code easier to understand by combining all project 'modules' under one directory.

- /
  - .meteor
  - .scripts
  - core/
  - docker/
  - apinf_packages/
    - apis/
    - branding/
    - catalog/
    - ...

  - tests/
- README.md
- ...

### PR DESCRIPTION
**Pull request by brylie**
Closes #2810 

# Changes
- move all APINF packages into `/apinf_packages` directory
- fix LESS / JavaScript imports
- fix ESLint errors

### CODE REVIEW

**Comment by marla-singer** [comment]
@brylie Change imports in `eslintignore` file, you've got ESlint error, look at Travic-CI log

**Comment by brylie** [comment]
@marla-singer done.

File: apinf_packages/settings/server/methods.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import { Accounts } from 'meteor/accounts-base';
 import { ServiceConfiguration } from 'meteor/service-configuration';
 
 // APInf imports
+// eslint-disable-next-line max-len
 import { githubSettingsValid, mailSettingsValid } from '/apinf_packages/core/helper_functions/validate_settings';
```
Comment (marla-singer) [inline apinf_packages/settings/server/methods.js]: @brylie Why did you add this rule here?
